nipype/interfaces/fsl/utils.py

```diff
@@ -1700,7 +1700,28 @@ def _gen_filename(self, name):
             return self._list_outputs()['out_file']
         return None
 
+      
+class FslOrientInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, desc='input file', argstr='%s', position=2, mandatory=True)
+    main_option = traits.Str(desc='main option', argstr='-%s', position=0, mandatory=True)
```

Comment:
Since this command takes a specific (and historically very stable) set of main options, it would be best to enumerate them. The current interface attempt [available in SAMRI](https://github.com/IBT-FMI/SAMRI/blob/6b38e29960e59115cb6e8ad0f1c23f4bcc3c9264/samri/pipelines/extra_interfaces.py#L529) demonstrates how.